src/store.js

```diff
@@ -4,7 +4,12 @@ export const store = {
         statusIndicator: "fine",
         foodIndicator: "100%",
         ageIndicator: 3,
-    }
+    },
+    indicators: {
+        state:  ["fine", "sick", "hungry", "dead"],
```

Comment:
if you stick with the array approach, you will have to do searching in the array every time you want to change the status. But what if more statuses will be added in the future? You will have to go through the entire app and redo the array indexes.
Objects give you flexibility of knowing the object key by its name ("sick", "hungry") from business requirements and if they change, you don't have to redo the indexes, you simply access the object key by state name, which almost never changes. You can also easily add new states in the future without changing other code.

Would be simpler and more efficient to use a simple object of states. Moreover, in such object you could reflect transitions between states, something like:
```javascript
const states: {
  fine: {
    uiText: 'Fine',
    nextStates: ['sick', 'hungry']
  },
  sick: {
    uiText: 'Sick',
    nextStates: ['dead'],
    prevStates: ['fine']
  },
}
```

Reply:
fixed. Commit 9805fc9